INSTALL.md

```diff
@@ -15,11 +15,11 @@ This guide assumes a fresh install of Ubuntu 22.04.
 
 ## Dependencies
 
-* HOL4 
-* Ott
+* HOL4
 * Petr4
 * Git
 * Python 3
```

Comment:
Isn't this optional also now that Ott is optional? Or where are you using Python except for post-processing Ott output?

Reply:
Fixed.